docs/ref/django-admin.txt

```diff
@@ -776,6 +776,12 @@ Enables fixing of migration conflicts.
 Allows naming the generated migration(s) instead of using a generated name. The
 name must be a valid Python :ref:`identifier <python:identifiers>`.
 
+.. django-admin-option:: --no-header
+
+.. versionadded:: 2.2
+
+Does not add header comments to generated migration file(s).
```

Comment:
"Generate migration files without Django version and timestamp header."

Reply:
Done